api/screening/screeningRoute.js

```diff
@@ -99,31 +142,30 @@ router
         screening.startDate = req.body.startDate;
       }
       await screening.save();
-      res.status(200).json({
+      return res.status(200).json({
         message: 'Screening updated successfully',
         screening,
       });
     } catch (e) {
-      res.status(400).send(e);
+      return res.status(400).send(e);
     }
   })
   // @route DELETE api/screenings/id
   // @description Delete a screening
-  // @access admin
+  // @access Admin
   .delete(authMiddleware, adminMiddleware, async (req, res) => {
```

Comment:
Should probably remove also all the orders and tickets connected to this screening and also send email to clients that the screening was cancelled.

Reply:
In our case it was only possible to remove connected tickets for now. Emails - done.